src/reactviews/pages/TableDesigner/designerMainPane.tsx

```diff
@@ -145,6 +145,41 @@ export const DesignerMainPane = () => {
         }
     }
 
+    function getSortedSchemaValues() {
```

Comment:
Can this be done in the controller so that it doesn't need to sort the schemas on every render?